docs/api-inference/tasks/text_generation.md

```diff
@@ -0,0 +1,193 @@
+## Text generation
+
+Generate text based on a prompt.
+
+<Tip>
+
+For more details about the `text-generation` task, check out its [dedicated page](https://huggingface.co/tasks/text-generation)! You will find examples and related materials.
```

Comment:
We should add some text that refers to chat_completion

Reply:
addressed in https://github.com/huggingface/hub-docs/pull/1393/commits/3eb85e8082a187a9c998d50fcaaff29df305b160